org.jellyfin.JellyfinServer.Plugin.IntelComputeRuntime.yml

```diff
@@ -0,0 +1,137 @@
+app-id: org.jellyfin.JellyfinServer.Plugin.IntelComputeRuntime
+branch: "1"
+runtime: org.jellyfin.JellyfinServer
+runtime-version: stable
+sdk: org.freedesktop.Sdk//22.08
+build-extension: true
+separate-locales: false
+appstream-compose: false
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm15
+build-options:
+  prefix: /app/extensions
```

Comment:
I don't think this is the right prefix. It should be `/app/extensions/IntelComputeRuntime`